### ISSUE TITLE
Transaction categories for stock takes

### ISSUE THREAD

**Issue by Chris-Petty** [openmsupply/mobile#445]
Description: Should add a selector in the stock take manager or editor that allows for selecting a transaction category

Comments: https://bugs.msupply.org.nz/view.php?id=6945

**Comment by joshxg** [comment]
@Chris-Petty If this is still something we want to do can you flesh this out? (I don't have a mantis log in :))

**Comment by Chris-Petty** [comment]
Going to close this 'cause clearly no one has been bothered much by its absence. @craigdrown requested this issue, though.

The mantis discussion is a bit hard to follow, but there is a design document thankfully:
https://wiki.sussol.net/doku.php/msupply:design:customers:master_transaction_categories

**Comment by craigdrown** [comment]
@Chris-Petty I think this is done.

**Comment by Chris-Petty** [comment]
It ain't

![image](https://user-images.githubusercontent.com/7684221/60777217-52992000-a185-11e9-8543-dbeab6a02c47.png)
